test/runner.js

```diff
@@ -1,56 +1,66 @@
-const config = require('./../lib/config');
-const benchpress = require('@angular/benchpress');
-const fs = require('fs-extra');
-const path = require('path');
+const config = require("./../lib/config");
```

Comment:
In places where the code changed, the quotes are changed from `'` to `"`. It would be nice to stick to one pattern for the sake of consistency in the entire project. I'd recommend changing all double quotes to single ones or leaving the double quotes but then updating all files to use them as well.

Reply:
I added prettier and ran it on all files so we don't need to think about such stuff :)